src/Aspire.Hosting.Docker/DockerComposeAspireDashboardResourceBuilderExtensions.cs

```diff
@@ -37,6 +37,9 @@ internal static IResourceBuilder<DockerComposeAspireDashboardResource> CreateDas
         return builder.CreateResourceBuilder(resource)
                       .WithImage("mcr.microsoft.com/dotnet/nightly/aspire-dashboard")
                       .WithHttpEndpoint(targetPort: 18888)
+                      // Expose the HTTP endpoint externally for the dashboard, it is password protected
+                      // and disabled by default so an explicit call is required to turn it on.
+                      .WithEndpoint("http", e => e.IsExternal = true)
                       .WithHttpEndpoint(name: "otlp-grpc", targetPort: 18889);
```

Comment:
Should this also expose `oltp-http` port on 18890? See https://aspire.dev/dashboard/configuration/#common-configuration

(there is also the MCP port, but I think its best to wait until that area stablizes)